### DISCUSSION TITLE
[Tests] Setup 'brakeman' gem

### DISCUSSION THREAD

**Discussion by rgenchev** [Terrastories/terrastories#729] [Temporarily Archived Tickets]
I think that it's a good idea to integrate `brakeman` (https://github.com/presidentbeef/brakeman) gem in the project. By installing it we could avoid many vulnerabilities.

@MxOliver, what do you think?

**Comment by auto-comment** [comment]
<img src="https://media.tenor.com/images/a44d0c84e5e535a649b16011b8d3c54c/tenor.gif" height=200 /> <br /> Thank you for raising a issue. We will try and get back to you as soon as possible. <br /> Please make sure you have given us as much context as possible. <br /> Best, <br /> @terrastories-stewards

**Comment by kalimar** [comment]
Sounds good to me. I'd like to document this so we remember to run it every so often. We may need a new doc for periodic app maintenance work and a log. I'd like to chat about that so we take proper advantage of tools like these.

**Comment by rgenchev** [comment]
We could integrate it into the CI. This way it'll run automatically on every commit.

**Comment by rgenchev** [comment]
Can I continue working on this? :)
